sonar-scanner-cli-entrypoint.yaml

```
@@ -0,0 +1,32 @@
+# Fetches the sonar-scanner-cli entrypoint script used in the upstream docker image. We
+# need this for our image, but as it's stored in a different repo vs the source
+# code, we need to fetch it and be able to auto-update.
+package:
+  name: sonar-scanner-cli-entrypoint
+  version: 10.0.3.1430_5.0.1
+  epoch: 0
+  description: Fetches the sonar-scanner-cli entrypoint script from upstream docker repository.
+  copyright:
+    - license: LGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://raw.githubusercontent.com/SonarSource/sonar-scanner-cli-docker/${{package.version}}/bin/entrypoint.sh
+      expected-sha256: d93310f2a96b766e0c96de33b519d0e414d863e28a433d2ca7832fcaa5926979
+      extract: false
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin/
+      chmod +x entrypoint.sh
+      cp entrypoint.sh ${{targets.destdir}}/usr/bin/
+
+update:
+  enabled: true
```

```suggestion
  manual: true
```

There's no obvious way with tags to automate updates.